src/containers/offerContainer/offerContainer.tsx

```diff
@@ -0,0 +1,46 @@
+import { FC } from 'react'
+import Box from '@mui/material/Box'
+import OfferCard from '~/components/offer-card/OfferCard'
+import { Grid } from '@mui/material'
+import { CardsViewTypes, OfferResponse } from '~/types'
+
+interface OfferContainerProps {
+  viewMode: CardsViewTypes
+  offerCards: OfferResponse[]
+}
+
+const OfferContainer: FC<OfferContainerProps> = ({ viewMode, offerCards }) => {
+  const onBookmarkClick = (id: string) => {
+    console.log(id)
+  }
+
+  const arrayOfCards = offerCards.map((el) => (
+    <Grid item key={el._id} md={4} sm={4} xs={2}>
```

Comment:
i guess, you don`t need md here, sm will be enough

Reply:
Fixed.